coordinator/src/node.rs

```diff
@@ -240,17 +251,27 @@ impl Node {
     #[autometrics]
     async fn open_position(
         &self,
+        conn: &mut PgConnection,
         trade_params: &TradeParams,
         fee_payment_hash: PaymentHash,
+        coordinator_leverage: f32,
     ) -> Result<()> {
         let peer_id = trade_params.pubkey;
         tracing::info!(%peer_id, ?trade_params, "Opening position");
 
         let margin_trader = margin_trader(trade_params);
-        let margin_coordinator = margin_coordinator(trade_params);
+        let margin_coordinator = margin_coordinator(trade_params, coordinator_leverage);
```

Comment:
:wrench: Shouldn't the coordinator leverage be added to the trade params then?

Reply:
I tend to agree and gave it a try. But it got very quickly very complicated, so I opted to leave it as is for now. But the whole `filled_with` and `trade_params` model probably need some rework anyways.